### PR TITLE
Add service account for watching prowjobs in aggregator

### PR DESCRIPTION
**Pull request by xueqzhan**
[TRT-1240](https://issues.redhat.com/browse/TRT-1240)

### CODE REVIEW

File: clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name: aggregator-waiter-secret-2023-10
+  namespace: trt-aggregator
+  annotations:
+    kubernetes.io/service-account.name: aggregator-waiter
+type: kubernetes.io/service-account-token
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: aggregator-waiter
```
Comment (dgoodwin) [inline clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml:27]: Can we name trt-aggregator-waiter

File: clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  namespace: ci
+rules:
+  - apiGroups:
+      - prow.k8s.io
+    resources:
+      - prowjobs
+    verbs:
+      - get
+      - list
+      - watch
```
Comment (dgoodwin) [inline clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml:37]: Lets see if Jakub is ok with this vs re-using prowjob_reader.

File: clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Service Accounts for TRT Aggregator Use
+    openshift.io/display-name: TRT CI Aggregator
+  name: trt-aggregator
```
Comment (dgoodwin) [inline clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml:7]: I wouldn't have thought an SA in one namespace could bind to a role in another, but if it works then that's good.

File: clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: aggregator-waiter
```
Comment (dgoodwin) [inline clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml:12]: trt-aggregator-watier

(for consistency)

**Review by jmguzik — APPROVED** [review]

**Pull request merged**